ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/authentication/JWTAuthOptionsFactory.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright ConsenSys AG.
+ * Copyright Hyperledger Besu Contributors
```

Comment:
Contributors? Is it interchangable with hyperledger besu or is there some rule I'm not aware of..

Reply:
I believe the rule is Contributors, since we technically hold the copyright, but when i looked in our wiki I couldn't easily find a ruling on it.